src/xdocs/filters/suppresswithplaintextcommentfilter.xml

```diff
@@ -312,6 +312,37 @@ CREATE TABLE STATION (
   LAT_N REAL,
   LONG_W REAL);
         </source>
+        <p id="Example9-config">
+            This check is not limited to comments. It can match any regular expression patterns as
+            delimiters (via offCommentFormat and onCommentFormat) for the suppression. The following
+            example shows how to suppress violations within a text block for LineLengthCheck.
+        </p>
+        <source>
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck&quot;&gt;
```

Comment:
`LineLength`

no reason to bother users with fully qualified names.

Reply:
Done